<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraExpertSettingsWidget</class>
 <widget class="QWidget" name="CameraExpertSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>682</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>16</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="iconLabel">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="generalWarningLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Do not change these settings unless you are absolutely sure of their potential impact on your system performance.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1058</width>
        <height>681</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
       <property name="spacing">
        <number>24</number>
       </property>
       <item>
        <widget class="QWidget" name="leftWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="spacing">
           <number>16</number>
          </property>
          <item>
           <widget class="QGroupBox" name="settingsGroupBox">
            <property name="title">
             <string>Camera Settings</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QCheckBox" name="settingsDisableControlCheckBox">
               <property name="text">
                <string>Keep camera stream and profile settings</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="settingsWarningLabel">
               <property name="text">
                <string>Quality and frame rate (FPS) settings in the Recording Schedule will become irrelevant.</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="bitratePerGopCheckBox">
               <property name="text">
                <string>Calculate bitrate per GOP instead of bitrate per second</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="bitrateIncreaseWarningLabel">
               <property name="text">
                <string>It will increase bitrate for camera.</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="secondStreamGroupBox">
            <property name="title">
             <string>Secondary Stream</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QCheckBox" name="secondStreamDisableCheckBox">
               <property name="text">
                <string>Disable secondary stream</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="secondStreamQualityWidget" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBoxMotionDetection">
            <property name="title">
             <string>Motion Detection</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <property name="sizeConstraint">
                <enum>QLayout::SetNoConstraint</enum>
               </property>
               <item>
                <widget class="QCheckBox" name="checkBoxForceMotionDetection">
                 <property name="text">
                  <string>Force motion detection for stream:</string>
                 </property>
                 <property name="tristate">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBoxForcedMotionStream"/>
               </item>
               <item>
                <spacer name="horizontalSpacer_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>1</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="rightWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>16</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBoxArchive">
            <property name="title">
             <string>Archive</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="checkBoxPrimaryRecorder">
               <property name="text">
                <string>Do not archive primary stream</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxSecondaryRecorder">
               <property name="text">
                <string>Do not archive secondary stream</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBoxRTP">
            <property name="title">
             <string>Media Streaming</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="labelTransport">
                 <property name="text">
                  <string>RTP transport:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBoxTransport">
                 <item>
                  <property name="text">
                   <string>Auto</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>TCP</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>UDP</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBoxPtzControl">
            <property name="title">
             <string>PTZ</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QCheckBox" name="checkBoxDisableNativePtzPresets">
               <property name="text">
                <string>Disable native presets, use system presets instead</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="settingsDisableNativePtzWarningLabel">
               <property name="text">
                <string>Warning! Native camera presets will not be avaliable.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="horizontalWidget_3" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <property name="margin">
                 <number>0</number>
                </property>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="logicalIdGroupBox">
            <property name="title">
             <string>Logical ID</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QSpinBox" name="logicalIdSpinBox">
                 <property name="maximum">
                  <number>999</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="generateLogicalIdButton">
                 <property name="text">
                  <string>Generate</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="resetLogicalIdButton">
                 <property name="text">
                  <string>Reset</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="InvalidLogicalIdText">
               <property name="text">
                <string>Invalid Logical Id label</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>290</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="restoreDefaultsButton">
       <property name="text">
        <string>Restore Defaults</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>settingsDisableControlCheckBox</tabstop>
  <tabstop>bitratePerGopCheckBox</tabstop>
  <tabstop>secondStreamDisableCheckBox</tabstop>
  <tabstop>checkBoxForceMotionDetection</tabstop>
  <tabstop>comboBoxForcedMotionStream</tabstop>
  <tabstop>checkBoxPrimaryRecorder</tabstop>
  <tabstop>checkBoxSecondaryRecorder</tabstop>
  <tabstop>comboBoxTransport</tabstop>
  <tabstop>checkBoxDisableNativePtzPresets</tabstop>
  <tabstop>restoreDefaultsButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
